src/core/Akka/Util/Base64Encoding.cs

```diff
@@ -6,12 +6,26 @@
 //-----------------------------------------------------------------------
 
 using System.Text;
+using Akka.Annotations;
+using Microsoft.Extensions.ObjectPool;
 
 namespace Akka.Util
 {
     /// <summary>
-    /// TBD
+    /// INTERNAL API.
+    ///
+    /// Used to help reduce memory allocations.
     /// </summary>
+    internal static class PooledObject
+    {
+        public static readonly ObjectPool<StringBuilder> StringBuilderPool =
+            new DefaultObjectPoolProvider().CreateStringBuilderPool(512, 2048);
```

Comment:
Are we expecting to re-use this pool elsewhere? These sizes may be a bit much otherwise.

Reply:
I think there's several other places where we can use it, but the sizes I chose are arbitrary

Reply:
I was mostly curious if there was significant overhead in accessing and returning a pooled `StringBuilder` vs Gen 0-ing one.